package/MDAnalysis/analysis/psa.py

```diff
@@ -338,7 +338,7 @@ def get_coord_axes(path):
 
 
 def hausdorff(P, Q):
-    r"""Calculate the Hausdorff distance between two paths.
+    r"""Calculate the undirected Hausdorff distance between two paths.
```

Comment:
replace "undirected" by "symmetric": this is what it is called in 

D. P. Huttenlocher, G. A. Klanderman, and W. J. Rucklidge. Comparing images using the hausdorff distance. IEEE Transactions on Pattern Analysis and Machine Intelligence, 15(9):850–863, 1993.

(and also in the PSA paper).